users/api_views.py

```diff
@@ -0,0 +1,15 @@
+from django.template import Context
+from rest_framework.generics import CreateAPIView
+
+from .models import User
+from .serializers import UserSerializer
+
+
+class UserApiRegistration(CreateAPIView):
```

Comment:
```suggestion
class UserAPIRegistration(CreateAPIView):
```
Let's keep naming consistent with Django.

Reply:
.